### ISSUE TITLE
php8.1-cli package missing

### ISSUE THREAD

**Issue by 2gen** [oerdnj/deb.sury.org#1815]
Debian 11 bullseye WSL

tldr: E: Failed to fetch https://packages.sury.org/php/pool/main/p/php8.1/php8.1-cli_8.1.7-2%2b0%7e20220625.22%2bdebian11%7e1.gbp4d1949_amd64.deb  404  Not Found [IP: 143.244.60.109 443]

APT OUTPUT
---------------
The following NEW packages will be installed:
  libapache2-mod-php8.1 php8.1 php8.1-cli php8.1-common php8.1-fpm php8.1-mysql php8.1-opcache php8.1-readline
0 upgraded, 8 newly installed, 0 to remove and 11 not upgraded.
Need to get 6082 kB of archives.
After this operation, 26.8 MB of additional disk space will be used.
Get:1 https://packages.sury.org/php bullseye/main amd64 php8.1-common amd64 8.1.7-2+0~20220625.22+debian11~1.gbp4d1949 [659 kB]
Get:2 https://packages.sury.org/php bullseye/main amd64 php8.1-opcache amd64 8.1.7-2+0~20220625.22+debian11~1.gbp4d1949 [336 kB]
Get:3 https://packages.sury.org/php bullseye/main amd64 php8.1-readline amd64 8.1.7-2+0~20220625.22+debian11~1.gbp4d1949 [12.6 kB]
Err:4 https://packages.sury.org/php bullseye/main amd64 php8.1-cli amd64 8.1.7-2+0~20220625.22+debian11~1.gbp4d1949
  404  Not Found [IP: 143.244.60.109 443]
Get:5 https://packages.sury.org/php bullseye/main amd64 libapache2-mod-php8.1 amd64 8.1.7-2+0~20220625.22+debian11~1.gbp4d1949 [1602 kB]
Get:6 https://packages.sury.org/php bullseye/main amd64 php8.1-fpm amd64 8.1.7-2+0~20220625.22+debian11~1.gbp4d1949 [1669 kB]
Get:7 https://packages.sury.org/php bullseye/main amd64 php8.1 all 8.1.7-2+0~20220625.22+debian11~1.gbp4d1949 [25.5 kB]
Get:8 https://packages.sury.org/php bullseye/main amd64 php8.1-mysql amd64 8.1.7-2+0~20220625.22+debian11~1.gbp4d1949 [116 kB]
Fetched 4421 kB in 1s (5691 kB/s)
E: Failed to fetch https://packages.sury.org/php/pool/main/p/php8.1/php8.1-cli_8.1.7-2%2b0%7e20220625.22%2bdebian11%7e1.gbp4d1949_amd64.deb  404  Not Found [IP: 143.244.60.109 443]

APT Cache
---------------
apt-cache policy php8.1-cli
php8.1-cli:
  Installed: (none)
  Candidate: 8.1.7-2+0~20220625.22+debian11~1.gbp4d1949
  Version table:
     8.1.7-2+0~20220625.22+debian11~1.gbp4d1949 500
        500 https://packages.sury.org/php bullseye/main amd64 Packages

**Comment by oerdnj** [comment]
Did you run `apt update` first?
